src/main/java/com/smartbear/postman/PostmanImporter.java

```diff
@@ -98,11 +98,12 @@ public PostmanImporter(TestCreator testCreator) {
 
     public WsdlProject importPostmanCollection(WorkspaceImpl workspace, String filePath) {
         WsdlProject project = null;
-        File jsonFile = new File(filePath);
         String postmanJson = null;
+        UrlClientLoader loader = new UrlClientLoader(filePath);
+        loader.setUseWorker(false);
         try {
-            postmanJson = FileUtils.readFileToString(jsonFile);
-        } catch (IOException e) {
+            postmanJson = IOUtils.toString(loader.load(), StandardCharsets.UTF_8);
```

Comment:
As a side effect of the call ``loader.load()`` we'll get the returned data to be cached (see ``urlCache.put(url, result);``). Do we really want them to stay in the cache?

Reply:
@Alexander-Sviridov I'm not sure if we need cached data, but it's standard way to load, is there other one, without caching data? Or do you want me to extend the load() methods to load, without caching?

Reply:
OK, I agree that it is a standard approach so let's keep it as is. But we should use ``ProgressDialog`` here instead of ``setCursor(Cursor.getPredefinedCursor(Cursor.WAIT_CURSOR))``.